pkg/processor/build/runtime/test/suite/suite.go

```diff
@@ -323,6 +324,16 @@ func (suite *TestSuite) compressAndDeployFunctionFromURL(archiveExtension string
 	compressor func(context.Context, io.Writer, []archiver.File) error) {
 
 	createFunctionOptions := suite.getDeployOptionsDir("reverser")
+	runAsUserID := int64(1000)
+	runAsGroupID := int64(2000)
+	fsGroup := int64(3000)
```

Comment:
Is this for all tests in this suite?
The default is running as root so it's better to have the majority of tests using the default

Reply:
@TomerShor only for archive tests. We run as root in all the other places as root is default.